samza-test/src/main/python/configs/tests.json

```diff
@@ -1,5 +1,5 @@
 {
-  "samza_executable": "samza-test_2.10-0.11.0-SNAPSHOT.tgz",
+  "samza_executable": "samza-test_2.10-0.10.1-SNAPSHOT.tgz",
```

Comment:
Makes sense. Let me sync w/ Xinyu Liu on the latest check-ins to master regarding to the version generated from trunk.

Meanwhile, I would encourage you to rebase again the latest master and validate that the version here still works. Thanks!